generators/entity/templates/src/test/java/package/web/rest/_EntityResourceIntTest.java

```diff
@@ -4,8 +4,8 @@
 import <%=packageName%>.<%= mainClass %>;
 import <%=packageName%>.domain.<%= entityClass %>;
 import <%=packageName%>.repository.<%= entityClass %>Repository;<% if (service != 'no') { %>
-import <%=packageName%>.service.<%= entityClass %>Service;<% } else { if (searchEngine == 'elasticsearch') { %>
```

Comment:
ok since you are using `<%= entityInstance %>SearchRepository.deleteAll();` anyway